### ISSUE TITLE
d3.append(node) is not working in d3.js, so needs to be changed to the function returning node

### ISSUE THREAD

**Issue by shamansir** [shamansir/rpd#439]
I can't believe I used `d3_tiny`'s `.append(node)` ability every time, but by [docs](https://github.com/d3/d3/blob/master/CHANGES.md#selections-d3-selection) it seems it is exactly what actually happened.

**Comment by shamansir** [comment]
The difference is that I usually use code like this:

```
var d3Selection = d3.select(...)...;
var anotherD3Selection = d3.select(...)...;
d3Selection.....append(anotherD3Selection.node());
```

but following to d3.js docs, it was wrong from the start (but so intuitive!)

**Comment by shamansir** [comment]
And so, relates to #405

**Comment by shamansir** [comment]
For now, I'm changing it to:

```javascript
var ƒ = Rpd.unit;

d3.....append(ƒ(d3().....node()));
```

But probably in some cases it would be convenient to use `selection.append('svg:text')` syntax.

**Comment by shamansir** [comment]
Fixed with #440
